lib/rules/no-array-prototype-extensions.js

```diff
@@ -160,33 +160,34 @@ function variableNameToWords(name) {
  * @param {String} [options.importedGetName] The name of the imported get specifier from @ember/object package
  * @returns {Object|[]}
  */
+/* eslint complexity: ["error", 50]*/
```

Comment:
We're going to hit this limit again eventually right? You could just disable the rule for this function.